erp/management/commands/import_centres_vaccination.py

```diff
@@ -29,77 +26,16 @@ def add_arguments(self, parser):
             action="store_true",
             help="Afficher les raisons d'écartement d'import de certains centres",
         )
-
-    def get_json(self, url):
-        try:
-            return requests.get(url).json()
-        except requests.exceptions.RequestException as err:
-            raise RuntimeError(
-                f"Erreur de récupération des données JSON: {url}:\n  {err}"
-            )
-
-    def retrieve_latest_dataset_url(self):
-        try:
-            json_data = self.get_json(self.api_url)
-            json_resources = [
-                resource
-                for resource in json_data["resources"]
-                if resource["format"] == "json"
-            ]
-            if len(json_resources) == 0:
-                raise RuntimeError("Jeu de donnée JSON abenst.")
-            return json_resources[0]["latest"]
-        except (KeyError, IndexError, ValueError) as err:
-            raise RuntimeError(
-                f"Impossible de parser les données depuis {self.api_url}:\n{err}"
-            )
-
-    def retrieve_json_data(self, dataset_url):
-        return self.get_json(dataset_url)
+        parser.add_argument(
+            "--report",
+            action="store_true",
+            help="Envoi un mail en fin d'opération",
+        )
 
     def handle(self, *args, **options):  # noqa
         self.stdout.write("Importation des centres de vaccination")
 
         try:
-            if "dataset-url" in options:
-                json_data = self.retrieve_json_data(options["dataset-url"])
-            else:
-                latest_dataset_url = self.retrieve_latest_dataset_url()
-                json_data = self.retrieve_json_data(latest_dataset_url)
+            import_centres_vaccination.job(dataset_url=options["dataset-url"], verbose=options["verbose"], report=options["report"])
```

Comment:
Nit: add a trailing comma so this spreads over multiple lines for better readability.